rke/k8s_rke_system_images.go

```diff
@@ -6174,6 +6218,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
 		},
+		// Enabled out of band post v2.5.12
+		"v1.20.15-rancher1-2": {
+			Etcd:                      "rancher/mirrored-coreos-etcd:v3.4.15-rancher1",
+			Kubernetes:                "rancher/hyperkube:v1.20.15-rancher1",
+			Alpine:                    "rancher/rke-tools:v0.1.79",
```

Comment:
rke-tools in 2.6 has been bumped to v0.1.80, either this needs to be updated here or 2.6 needs a copy of this k8s version